releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,19 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.1.0 - xx xx, 2023
+
+### Enhancements
+* Added the `auth.workloadIdentity.managedIdentityClientIdReference` property for workloadIdentity authentication. It enables the retrieval of the client ID of a user-assigned managed identity from a ConfigMap. [#812](https://github.com/Azure/AppConfiguration/issues/812)

```

Comment:
just to confirm, do we only support user-assigned managed identity? what about system-assigned or service principal?

Reply:
For now, we only support `ManagedIdentityClientIdReference` for workload identity, which is asked by the customer. If system-assigned managed identity is used for auth, users don's need to set `spec.auth` filed. And we have `servicePrincipalReference` for service principal. We could add configMapReference for `auth` later if there's more voices ask for it.